js/ui/button_group.js

```diff
@@ -0,0 +1,246 @@
+            * @type Enums.ButtonGroupSelectionMode
+            * @default 'single'
+            */
+            selectionMode: "single",
+
+            /**
+             * @name dxButtonGroupOptions.selectedItems
+             * @type Array<any>
+             * @fires dxButtonGroupOptions.onSelectionChanged
+             */
+            selectedItems: [],
+
+            /**
+             * @name dxButtonGroupOptions.selectedItemKeys
+             * @type Array<any>
+             * @fires dxButtonGroupOptions.onSelectionChanged
+             */
+            selectedItemKeys: [],
+
+            /**
+             * @name dxButtonGroupOptions.keyExpr
+             * @type string|function
+             * @default 'text'
+             */
+            keyExpr: "text",
+
+            /**
+             * @name dxButtonGroupOptions.items
+             * @type Array<ButtonGroupItem>
+             * @inherits CollectionWidgetItemTemplate
+             */
+            items: [],
+
+            /**
+             * @name dxButtonGroupOptions.itemTemplate
+             * @type template|function
+             * @default "item"
+             * @type_function_param1 itemData:object
+             * @type_function_param2 itemIndex:number
+             * @type_function_param3 itemElement:dxElement
+             * @type_function_return string|Node|jQuery
+             */
+            itemTemplate: "item",
+
+            /**
+             * @name dxButtonGroupOptions.onSelectionChanged
+             * @extends Action
+             * @type function(e)
+             * @type_function_param1 e:object
+             * @type_function_param1_field4 addedItems:array<any>
+             * @type_function_param1_field5 removedItems:array<any>
+             * @action
+             */
+            onSelectionChanged: null
+        });
+    },
+
+    _initTemplates() {
+        this.callBase();
+
+        /**
+         * @name dxButtonGroupItem
+         * @inherits CollectionWidgetItemTemplate
+         * @type object
+         */
+        /**
+         * @name dxButtonGroupItem.hint
+         * @type String
+         */
+        /**
+         * @name dxButtonGroupItem.type
+         * @type Enums.ButtonType
+         * @default 'normal'
+         */
+        /**
+         * @name dxButtonGroupItem.icon
+         * @type String
+         */
+        /**
+         * @name dxButtonGroupItem.html
+         * @hidden
+         */
+        this._defaultTemplates["item"] = new BindableTemplate((($container, data) => {
+            const itemIndex = $container.data("dxItemIndex");
+            itemIndex === 0 && $container.addClass(BUTTON_GROUP_FIRST_ITEM_CLASS);
+            this._createComponent($container, Button, extend({}, data, this._getBasicButtonOptions()));
+        }), ["text", "type", "icon", "disabled", "visible", "hint"], this.option("integrationOptions.watchMethod"));
+    },
+
+    _initMarkup() {
+        this.setAria("role", "buttongroup");
```

Comment:
It's an unacceptable role.
See more [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques)